### PR TITLE
Added info about allowlisting devices (TCD-2283)

### PR DESCRIPTION
**Pull request by lauramthomas**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by lysannep — APPROVED** [review]

**Review by aoleszyn — APPROVED** [review]

File: docs/mobile-apps/supported-devices.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ This is dedicated pool of devices just for your organization. On the mobile devi
 * You need specific settings which are set on the devices all the time.
 * You need to establish a secure [IPSec VPN connection](/secure-connections/ipsec-vpn) between your network and the Sauce Labs cloud.
 
+#### Allowlisting Apps and Accounts
+* Apps installed on the private device can be selected, by name, to be preserved between sessions. These apps and their data will not be cleaned.
```
Comment (wswebcreation) [inline docs/mobile-apps/supported-devices.md:88]:
```suggestion
* Apps installed on the private device can be selected, by Android `packageName` or iOS `bundleId`, to be preserved between sessions. These apps and their data will not be cleaned.
```

File: docs/mobile-apps/supported-devices.md [hunk 2]
```diff
@@ -84,6 +84,11 @@ This is dedicated pool of devices just for your organization. On the mobile devi
...
+* Google accounts signed into the Play Store can be preserved between sessions.
```
Comment (wswebcreation) [inline docs/mobile-apps/supported-devices.md:89]: Not sure if this is correct / linked to private devices

File: docs/mobile-apps/supported-devices.md [hunk 3]
```diff
@@ -84,6 +84,11 @@ This is dedicated pool of devices just for your organization. On the mobile devi
...
+* Apple IDs signed in to the device and App Store can be preserved between sessions.
```
Comment (wswebcreation) [inline docs/mobile-apps/supported-devices.md:90]: Same here, not sure if this is correct / linked to private devices

**Review by wswebcreation — CHANGES_REQUESTED** [review]
Hi @lauramthomas 

I've added some comments, maybe also check with @kristofmuhi regarding the iCloud and Google accounts to be sure that they are not removed during a cleaning process

**Pull request closed without merging**